lab-bundle-spec.md

```diff
@@ -162,7 +162,7 @@ fleet                         |          | enum*   | Specify a Qwiklabs fleet to
   id: secondary_project
   fleet: gcpfree
   dm_template:
-    script: deployment_manager/instance_pool.yaml
+    script: deployment_manager
```

Comment:
I find these 2 examples confusing because one has locales under script, one doesn't

Reply:
Also existing labs do not use 'dm_template' at all. They all use 'dm_script' field